data/json/items/ammo/cannon.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "id": "cannon_shot",
+    "type": "AMMO",
+    "name": { "str": "cannon round shot" },
+    "description": "A bundle of wadding, black gunpowder, and cannonball, ready to be loaded into a cannon",
+    "weight": "4800 g",
+    "volume": "2500 ml",
```

Comment:
4.3 kg of round shot has a volume of about 540 mL; 0.5 kg of wadding and gunpowder has a density around that of water and a volume around 500 mL.  I'd approximate the volume as 1000 mL or maybe 1500 mL at most; 2500 mL seems high (but I will be happy to be overruled by an actual cite to a reference!)